cluster-provision/README.md

```diff
@@ -17,7 +17,7 @@
 * `kubevirtci/cli`: `sha256:1dd015dea4f12e6dcb8e31be3eeb677fed96f290ef4a4892a33c43d666053536`
 * `kubevirtci/gocli`: `sha256:caf1c3f63dc5f3137795084e492a48c802a7ef2e7f7fbe1d67e5cff684d376a3`
 * `kubevirtci/base`: `sha256:034de1a154409d87498050ccc281d398ce1a0fed32efdbd66d2041a99a46b322`
-* `kubevirtci/centos:1804_02`: `sha256:70653d952edfb8002ab8efe9581d01960ccf21bb965a9b4de4775c8fbceaab39`
+* `kubevirtci/centos:1905_01`: `sha256:f2b204a3fabf71494c3cf7a145dae88e63f068a84ca10149b0aec523045dc2c1`
```

Comment:
Looks like 1905 is a pre-release build of upcoming centos 8. I think we should upgrade to 1902 instead. Rationale:
https://app.vagrantup.com/centos/boxes/7/versions/1902.01

Reply:
I think the documention was outdated, 1905 is announces in the meanwhile :)
https://blog.centos.org/2019/07/updated-centos-vagrant-images-available-v1905-01/
https://app.vagrantup.com/centos/boxes/7/versions/1905.1

Reply:
@slintes yep, good point. Good for us!